src/service/url-replacements-impl.js

```diff
@@ -785,9 +785,10 @@ export class UrlReplacements {
       'CLIENT_ID': true,
       'QUERY_PARAM': true,
     };
+    const additionalURLParameters = element.getAttribute('data-amp-addparams');
```

Comment:
Should have caught this last time. Please rename to `additionalUrlParameters` (the `Url` part).